draft-ietf-tls-deprecate-obsolete-kex.md

```diff
@@ -215,26 +206,27 @@ of implementing security countermeasures correctly.
 
 At a rough glance, the problems affecting FFDHE in TLS 1.2 are as follows:
 
-1. FFDHE suffers from interoperability problems because there is no mechanism
-for negotiating the group size, and some implementations only support small
-group sizes (see {{!RFC7919}}, Section 1).
+1. FFDHE suffers from interoperability problems because there is no mechanism for
+negotiating the group, and some implementations only support small group sizes
+(see {{!RFC7919}}, Section 1).
 
-2. In practice, some operators use 1024-bit FFDHE groups since this is the
+2. FFDHE groups may have small subgroups, which enables several attacks
+{{subgroups}}. When presented with a custom, non-standardized FFDHE group, a handshaking client cannot practically verify that the group chosen by the server does not suffer from this problem. There is also no mechanism for such handshakes to fall back to other key exchange parameters that are acceptable to the client.
+Custom FFDHE groups are widespread (as a result of advice based on {{weak-dh}}). 
```

Comment:
Maybe explicitly state why this is bad? Something like "Custom FFDHE groups are widespread (as a result of advice based on {{weak-dh}}), which means that clients cannot simply reject handshakes that present custom and thus potentially dangerous groups."

Reply:
Done, thanks :-)